webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/DataStructures.scala

```diff
@@ -219,6 +219,13 @@ final case class ParameterEntry(
     escapedValue:  String,
     agentType:     AgentType
 ) {
+
+  val escapedParameterName = parameterName.replaceAll("""[^\p{Alnum}_]""", "_")
+  // returns the escaped name of the parameter
+  def getEscapedParameterName(): String = {
```

Comment:
this is never used ?

Reply:
It's in https://github.com/Normation/rudder-techniques/pull/1868

Reply:
It was explained by the comment above that we need this function for string template